src/main/java/calc/Calc.java

```diff
@@ -0,0 +1,82 @@
+package calc;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calc {
+
+    private final static String CUSTOM_PATTERN = "//(.)\n(.*)";
+    private final static String COMMON_PATTERN = ",|:";
```

Comment:
```suggestion
    private static final String CUSTOM_PATTERN = "//(.)\n(.*)";
    private static final String COMMON_PATTERN = ",|:";
```

컨벤션 상 static이 final보다 우선입니다.